codespell.txt

```diff
@@ -0,0 +1,17 @@
+ba
```

Comment:
Is this file necessary? It would be better to skip `docs/es` instead of putting Spanish words in the ignore list.

Reply:
Yes the file is necessary but you are correct that we don't need Spanish words in the ignore list.

We use all lowercase "words" in the ignore list.  I just now ran:

```
codespell --skip="./docs/es,./docs/zh,./.git,./**/*.crt,./**/go.sum" | cut -d' ' -f2 | sort | uniq        
WithT
hel
nD
nd
nin
witht
```